src/content/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data.mdx

```diff
@@ -10,34 +10,59 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-import dashboardsDashboardMenu from 'images/dashboards_screenshot-crop_dashboard-menu.webp'
+You can use [dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) to visualalize and track your system's health, tackle issues, and plan any further steps. Learn to share all that information with your team, customers, or other stakeholders with various import and export options.
 
+## Import dashboards
 
-[Dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/) is a powerful tool to visualize and track how your system is doing, and allows you to tackle issues and plan next steps. Learn to share all that information with your team, customers, or other stakeholders, using the following options to import, share and export dashboards, or add and share charts (or other content) in a dashboard.
+At the [dashboards index](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#get-started), you can **[import a dashboard as JSON](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-import)**.
 
-## Import, share, and export dashboards [#dashboards]
+## Export dashboards 
 
-At the [dashboards index](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#get-started), you can **[import a dashboard as JSON](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-import)**.
 
-From any dashboard, share it with one of these options:
+You can export your dashboard as either a comma-seperated file(CSV) or a PDF. 
 
-* Copy the dashboard's permalink.
-* Copy your dashboard **as JSON** and add it to the clipboard by clicking on the **&lt;/>** icon on the right corner.
-* Export your dashboard **as a pdf file** clicking the <Icon name="fe-download"/> icon.
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Export CSV
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Export PDF
+        </TabsBarItem>
+    </TabsBar>
 
-<img
-  title="dashboard_menu.png"
-  alt="Dashboard menu.png"
-  src={dashboardsDashboardMenu}
-/>
+    <TabsPages>
+        <TabsPageItem id="1">
 
-You can also [duplicate your dashboard](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/#dashboards-clone).
+From any dashboard, export charts as CSVs by clicking the dropdown in the top right of a chart and selecting **Export as CSV**. This will download your data as a CSV. 
+
+While powerful, exporting as a CSV has limitations. A few things to keep in mind:
+
+- Data always exports in UTC time (-7:00)
+- Export doesn't support aggregator functions like `average()`
+- Export doesn't support the `COMPARE_WITH` clause
```

Comment:
Is there a way to also include formatting limitations? 
> The timestamp for format only is applied to `Timestamp` column (case sensitive)

And also about the apdex 

>`apdex` function with `FACET` clause returns the raw [Apdex counts](https://docs.newrelic.com/docs/apm/new-relic-apm/apdex/apdex-measure-user-satisfaction/#apdex-counts).

Both issues are quite common so users usually have complaints about it.